spec/granite/querying/query_builder_spec.cr

```diff
@@ -3,6 +3,22 @@ require "../../spec_helper"
 describe Granite::Query::BuilderMethods do
   describe "#where" do
     describe "with array argument" do
+      it "errors queries ids fields" do
```

Comment:
description of this test is confusing.  consider `it "correctly queries all rows with a list of id values" do`.

Reply:
changed